modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductProductRelations.testcase

```diff
@@ -55,6 +55,53 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11085. Can delete a product related to another product"
+	@priority = 3
+	test CanDeleteProductWithRelations {
+		property portal.acceptance = "false";
+
+		task ("Given a Minium Site is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And a product is related to another product") {
+			CommerceProducts.openProductsAdmin();
+
+			CommerceProducts.searchAndGoToProductEntry(entryProductName = "ABS Sensor");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Product Relations");
+
+			CommerceRelations.addProductRelation(productNameRelation = "U-Joint");
+
+			AssertElementPresent(
+				key_productName = "U-Joint",
+				key_value = "Up-Sell",
+				locator1 = "CommerceEntry#VALUE_PRODUCT_RELATIONS");
+		}
+
+		task ("When the admin delete the related product") {
+			CommerceProducts.openProductsAdmin();
+
+			CommerceNavigator.searchEntry(entryName = "U-Joint");
+
+			Click.clickNoMouseOverNoWaitForVisible(
+				key_productName = "U-Joint",
+				locator1 = "CommerceEntry#PRODUCT_ENTRY_CHECKBOX");
+
+			AssertElementPresent(locator1 = "Button#TRASH_ENABLED");
+		}
+
+		task ("Then the product is successfully deleted") {
+			ClickNoError(locator1 = "Button#TRASH_ENABLED");
+
+			CommerceNavigator.searchEntry(entryName = "U-Joint");
+
+			AssertElementNotPresent(
+				key_entryName = "ABS Sensor",
```

Comment:
You deleted U-Joint but made an assertion for ABS Sensor, how did the test pass on your local?